src/migrations/hotbar-store/5.0.0-beta.5.ts

```diff
@@ -20,7 +20,7 @@
  */
 
 import type { Hotbar } from "../../common/hotbar-store";
-import { catalogEntityRegistry } from "../../renderer/api/catalog-entity-registry";
+import { catalogEntityRegistry } from "../../main/catalog";
```

Comment:
This was just wrong since the `renderer` migration would never fire because the main code would have already done the migration before hand. And thus this migration was removing all items from all hotbars.